LPS-83379 Enabling auto tagging for Shuyang

4lejandrito opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

4lejandrito commented:
Hi @shuyangzhou,

As you asked in the email, I have removed tensorflow from portal and added it as a compileInclude dependency in the module. I reproduce the issue following these steps:

1. cd to `modules/apps/document-library/document-library-asset-auto-tagger-tensorflow`.
2. Run `gradle deploy`.
3. Wait until the module is STARTED.
4. Run `gradle clean deploy`.
5. Wait for a bit.
6. The server crashes displaying the following message:
      ```
      2018-07-24 14:34:25.110171: F tensorflow/core/lib/monitoring/collection_registry.cc:77] Cannot register 2 metrics with the same name: /tensorflow/cc/saved_model/load_attempt_count
      ```

Please let me know if you need help reproducing it.

Thanks for your help!

shuyangzhou commented:
See https://github.com/brianchandotcom/liferay-portal/pull/61595
